### PR TITLE
Import Game Teams data

### PR DESCRIPTION
**Pull request by AngelaGuardia**
**Additions:**

- Created GameTeam and GameTeamTest classes
- Created GameTeamCollection and GameTeamCollectionTest classes

**Changes made:**
- Deleted the extra line that was included in the game_teams_fixture files so that it only has 20 now.
- Refactored the setup methods for **all ______CollectionTest** classes to use class variables instead. This minimized the amount of times we're reading from our data files.
- Refactored the initialize methods for the TeamCollection and GameCollection classes to store the collections in a static array. This also minimizes the amount of times we're reading data from our files.

### CODE REVIEW

File: lib/team_collection.rb [hunk 1]
```diff
@@ -2,18 +2,17 @@
 require './lib/team'
 
 class TeamCollection
-  attr_reader :teams
+  attr_reader :all_teams
 
   def initialize(teams_path)
     @teams_csv = teams_path
-    @team_collection = all_teams
+    @all_teams = []
```
Comment (leahriffell) [inline lib/team_collection.rb:9]: @guardiac I like the renaming from collection.

**Review by leahriffell — APPROVED** [review]
Looks good! Thanks for being thoughtful about reducing the number of times we are reading the data.

File: lib/game_collection.rb [hunk 2]
```diff
@@ -2,19 +2,18 @@
...
     @game_path = './data/games_fixture.csv'
-    @game_collection = all_games
+    @all_games = []
+    create_games
   end
 
-  def all_games 
-    all_games = []
-    csv = CSV.foreach(@game_path, headers: true, converters: :numeric, header_converters: :symbol) do |row|
+  def create_games
```
Comment (sarynm12) [inline lib/game_collection.rb:13]: Love this. It does make sense to throw create_games in initialize. Good call

**Review by sarynm12 — APPROVED** [review]

File: lib/game_team.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    @result = data[:result]
+    @settled_in = data[:settled_in]
+    @head_coach = data[:head_coach]
+    @goals = data[:goals]
+    @shots = data[:shots]
+    @tackles = data[:tackles]
+    @pim = data[:pim]
+    @powerplay_opportunities = data[:powerplayopportunities]
+    @powerplay_goals = data[:powerplaygoals]
+    @faceoff_win_percentage = data[:faceoffwinpercentage]
```
Comment (hopesgit) [inline lib/game_team.rb:32]: shouldn't the symbols on these lines have underscores in them?

Comment (AngelaGuardia) [reply lib/game_team.rb:32]: No! The hash that I'm referencing doesn't include underscores (it's how they gave us the data). I added the underscores in the variable names to adhere to ruby convention! Give it a shot in pry and you should see what I mean.

**Review by hopesgit — APPROVED** [review]

**Pull request merged**